ITERATION NUMBER: 1
Current best mechanism: 
 [[-1  1  0  0  1]
 [ 0  1  0  1 -1]
 [ 0  1  1 -1  0]]
Current AIC value: -172.68875186791755
Previous best mechanism: 
 ['Nothing']
Previous AIC value: 1e+99
ITERATION NUMBER: 2
Current best mechanism: 
 [[-1  1  0  0  0  1]
 [ 0  1  0  0  1 -1]
 [ 0  0  0  1 -1  0]
 [ 0  1  1 -1  0  0]]
Current AIC value: -189.1133446801618
Previous best mechanism: 
 [[-1  1  0  0  1]
 [ 0  1  0  1 -1]
 [ 0  1  1 -1  0]]
Previous AIC value: -172.68875186791755
ITERATION NUMBER: 3
Current best mechanism: 
 [[-1  1  0  0  0  0  1]
 [ 0  0  0  0  1  1 -1]
 [ 0  1  0  0  0 -1  0]
 [ 0  0  0  1 -1  0  0]
 [ 0  1  1 -1  0  0  0]]
Current AIC value: -169.02676121401134
Previous best mechanism: 
 [[-1  1  0  0  0  1]
 [ 0  1  0  0  1 -1]
 [ 0  0  0  1 -1  0]
 [ 0  1  1 -1  0  0]]
Previous AIC value: -189.1133446801618
Task was destroyed but it is pending!
task: <Task pending name='Task-3' coro=<Kernel.poll_control_queue() running at /Users/md1621/anaconda3/lib/python3.10/site-packages/ipykernel/kernelbase.py:309> wait_for=<Future pending cb=[Task.task_wakeup()]> cb=[_chain_future.<locals>._call_set_state() at /Users/md1621/anaconda3/lib/python3.10/asyncio/futures.py:392]>
